### DISCUSSION TITLE
So, mastodon.social wants to redirect users to joinmastodon?

### DISCUSSION THREAD

**Discussion by ProcrastinatingEntrepreneur** [mastodon/mastodon#19811] [General]
I made an account on the c.im Server initially and that’s apparently down, and now I try to join mastodon.social and they apparently don’t want new users as they decided to reroute to the joinmastodon site. This is really putting me off of mastodon and I feel will kill the average users motivation to even bother themselves with switching. I just don’t see this being a good move. If you can’t handle it Server wise, put a queue up. The perspective this is putting out is only “important” people can join this “exclusive” server, which in my opinion just tramples over the decentralized, “for the people” persona you guys have been playing.

**Comment by tenorune** [comment]
There is nothing specifically special about having a mastodon.social account. 

What could be clearer than the text shown after you sign up?  It shows this and then a CTA to choose a server:

"Creating an account on  mastodon.social is currently not possible, but please keep in mind that you do not need an account on mastodon.social to use Mastodon."

**Reply by ProcrastinatingEntrepreneur** [reply]
While that’s a good looking excuse to pull on people and many sheep will fall for it, but that is most certainly a lie.  The special thing about it is the somewhat assured continuation of a server and not having to worry about signing up with a server that is gone months later. My c.Im account is pretty worthless apparently now that the server is down as the information isn’t allowing me to even login anywhere else on mastodon like it seems to state.

**Reply by puyoxyz** [reply]
> While that’s a good looking excuse to pull on people and many sheep will fall for it, but that is most certainly a lie. The special thing about it is the somewhat assured continuation of a server and not having to worry about signing up with a server that is gone months later. My c.Im account is pretty worthless apparently now that the server is down as the information isn’t allowing me to even login anywhere else on mastodon like it seems to state.

Mastodon.social isn't the only big server...

**Comment by Tslmuun** [comment]
c.im has resumed access.
